packages/financial-templates-lib/src/logger/Logger.ts

```diff
@@ -52,8 +52,10 @@ export function createNewLogger(
   transportsConfig = {},
   botIdentifier = process.env.BOT_IDENTIFIER || "NO_BOT_ID"
 ): AugmentedLogger {
+  const sessionId = Date.now();
   const logger = winston.createLogger({
     level: "debug",
+    defaultMeta: { sessionId },
```

Comment:
OOC, do you know which transports expose this defaultMeta? For instance, will it show up in slack and pagerduty or just the GCP console logs?

Reply:
I think it replaces part of the metadata field? https://github.com/winstonjs/winston#usage see the `defaultMeta` here. in any case we should add a comment here to say that this is a core part of Winston that we are overriding.

Reply:
are we overriding someting though @chrismaree ? We're just adding a field right?